.eslint-dictionary.json

```diff
@@ -1,5 +1,8 @@
 [
+  "75pk49boud2olipfda0ke3snic",
```

Comment:
t appears that a large portion of the additions are mocked values in tests. Perhaps the dictionary lint rules can be ignored for test files?